src/libraries/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Nit: Unnecessary edit

Reply:
VS did something that I cannot see or undo in the editor. I will revert the whole file in git.